src/components/file/file.tsx

```diff
@@ -143,26 +143,26 @@ export class File {
 
         return [
             <input
+                hidden={true}
                 id={this.fileInputId}
-                type="file"
                 onChange={this.handleFileChange}
-                hidden={true}
+                type="file"
             />,
             <limel-chip-set
                 disabled={this.disabled}
                 label={this.label}
-                required={this.required}
-                onKeyDown={this.handleKeyDown}
-                onKeyUp={this.handleKeyUp}
-                type="input"
-                value={chipArray}
-                onClick={this.handleFileSelection}
+                leadingIcon="upload_to_cloud"
                 onChange={this.handleChipSetChange}
-                onInteract={this.preventAndStop}
-                onDrop={this.handleFileDrop}
+                onClick={this.handleFileSelection}
                 onDragEnter={this.preventAndStop}
                 onDragOver={this.preventAndStop}
-                leadingIcon="upload_to_cloud"
+                onDrop={this.handleFileDrop}
+                onInteract={this.preventAndStop}
+                onKeyDown={this.handleKeyDown}
+                onKeyUp={this.handleKeyUp}
+                required={this.required}
+                type="input"
+                value={this.chipArray}
```

Comment:
At this point, this should still be `value={this.chipArray}`. It should only change in the next commit.

Sorry for being a pain in the butt…

Reply:
I'm aware of that but couldn't get VSC to cooperate and hoped you wouldn't catch it :D

Reply:
I can spend another 15 minutes to fix it if you really want.

Reply:
I really do. But I can fix it for you 😄

Reply:
Done 😉